tests/zfs-tests/include/libtest.shlib

```diff
@@ -4173,6 +4173,24 @@ function get_arcstat # stat
 	esac
 }
 
+#
+# Wait for the specified arcstat to reach a non-zero plateau.
+#
+function arcstat_plateau # stat
+{
+	typeset stat=$1
+	typeset do_once=true
+
+	while $do_once || [ $stat1 -ne $stat2 ] || [ $stat2 -eq 0 ]; do
+		typeset stat1=$(get_arcstat $stat)
+		sleep 2
+		typeset stat2=$(get_arcstat $stat)
+		do_once=false
+	done
+
+	echo $stat2
+}
```

Comment:
Since there's no upper bound on how long this helper waits for a settle, should it echo something like 'waiting for $stat to plateau...' when it starts, to make it easy to see what happened when a test times out?

Reply:
I guess that's in conflict with being able to use the interpolated output of the helper.  I'd like to see a wrapper for standalone calls as a separate thing from one used for interpolation, but it's not a biggie.

Reply:
I can easily fix this by inserting echoes right before the call to the helper in the tests.

Reply:
Or I can implement another helper like arcstat_quiescence_echo.